backend/node/base.py

```
@@ -29,8 +29,22 @@
 from .types import Address
 
 
+def _ensure_dotenv_loaded_for_chain_id() -> None:
+    if os.getenv("HARDHAT_CHAIN_ID") is not None:
+        return
+
+    try:
+        from dotenv import load_dotenv
+    except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)